### DISCUSSION TITLE
I don't want the list

### DISCUSSION THREAD

**Discussion by wmjordan** [community/community#12597] [Other Feature Feedback, Questions, & Ideas]
I don't want it. I hope it can be removed from my stars page.

I also hope that the starred repositories can be sorted by the time when they are recently updated.

EDIT:
The sort preference should be saved.

**Comment by jonathanp0** [comment]
That's already possible - select "Sort By: Recently Active"

**Reply by wmjordan** [reply]
> That's already possible - select "Sort By: Recently Active"

Thank you for the reply. Yes, it works this time, and next time when I return to that page, I have got to do that again.

**Comment by MindTooth** [comment]
Yes.  Having that gigant empty space taken up by nothing is a bit annoying on an iPad.

**Reply by wmjordan** [reply]
> Yes. Having that gigant empty space taken up by nothing is a bit annoying on an iPad.

Finally, I configured my Ad blocker to get rid of it.
